documentation/src/main/wiki/GatewayMode.md

```diff
@@ -0,0 +1,41 @@
+# Gateway Mode
+
+The gateway mode provides an alternative way of processing requests, alternative to than presented in [[Knot Routing section|KnotRouting]].
```

Comment:
> The gateway mode provides an alternative way of processing requests, alternative to than presented in [[Knot Routing section|KnotRouting]].

Some repetition and awkward grammar here. How about rewriting this as follows:

The gateway mode provides a way of processing requests alternative to that presented in the [[Knot Routing section|KnotRouting]].